Add organizing section (#71)
techworkersco/twc-site-nl: content/resources/organize/index.en.md

## Why organizing works

Organizing at your workplace is effective. Why? Because businesses need workers for their survival.

Only you, the workers, actually get things done for the business. You make the goods. You deliver the services. You produce value for the organization. Bosses, managers, and shareholders don't produce value. They just _extract_ the value created by you, the worker.

Without workers, nothing gets done. No goods get made. No service is delivered. Without goods or services, there's no profit. And without profit, the business collapses.

That means that you and your coworkers can be immensely powerful... _if you stick together and organize!_

Organize, and you can win the workplace changes you want.

<!-- [TODO: Add examples of workplace organizing that achieved successes!] -->

## Organizing foundations

### Find your drive

What would you want to change about your work? Why do you want it? Know your reasons. Know you deserve it, and that you’re worth it. Get motivated.

Your reasons for organizing will sustain you through any hard times.

### Connect with coworkers

> Meet up socially with your coworkers. Hold informal coffee chats, set up a memes channel, do games nights and recipe swaps. This way you build trust as the foundation from which to organize.

Author: Yes great point, thanks!

### Form a planning team

Find a couple of coworkers who also want to organize. Check that they have the time and energy to do so. You're the planning team now!

> Select a regular time outside work to meet up with your planning team. This helps you build solidarity and strategize together.

Author: Perfect 👌

- Are they already on board with organizing?
- Sentiment towards unions
- Whether they're a leader (see: Step 2)
- Who they could bring on board
- Last time contacted
- Contact details

<!-- TO DO: Create a doc or PDF that people can directly copy or download -->

Update the map every week.

> 💡 Keep your workplace map strictly inside your planning team! If the map leaks out, management is likely going to use it to sabotage your efforts.

### Step 2: Identify leaders

Identify the people that others listen to. This could be for any reason at all – maybe they’re respected for their work, but maybe they’ve just got great memes in the company Slack. Whatever the reason, these are the people you want on board. If you can recruit them, they’ll recruit others.

### Step 3: Talk with more coworkers

Talk with more of your coworkers about what frustrates them. Do they want more flexibility? Higher pay? Better safety standards? Three colleagues sitting in a room together complaining about management are halfway there.

Getting coworkers fired up is mostly listening and asking some questions here and there. Once people have identified some problems, ask how they’d change the situation.

> > 💡 Take all conversations about organizing off of work computers and channels as soon as possible---for example company email, Slack, Microsoft Teams, or Google Meet. Work channels are likely monitored, and any device that you got from work may be bugged with tracking software.

Reviewer: ```suggestion
> 💡 Take all conversations about organizing off of work computers and channels as soon as possible---for example company email, Slack, Microsoft Teams, or Google Meet. Work channels can sometimes be monitored, so be careful!
```

Just a suggestion. I agree with the sentiment, but it can sound a little tinfoil hat wearing. I think the bigger the company the less it's tinfoil and just a regular hat, so it's maybe a double edged sword.

Author: Ha yes true :D I think I'd like to keep the reference to work devices in there, but for sure it can be formulated a bit more down to earth, will do that

Once you've got more of your coworkers on board, you can start with actions and campaigns to build toward getting the workplace changes you want.

> 

Reviewer: ```suggestion

### Tips

Try to remove the words "we", "I", and "us" from your vocabulary. Organizing is for you and your co-workers working for better work conditions for you and your co-workers. At an extreme, remove the word thank you, because the thanking can be done when the job is done and you have the conditions you want for you and your co-workers.
```

The last part maybe a little cheesy, but the first part I think was a new/interesting learning from O4P for me.
